src/helpers/analytics/bindEvents.ts

```diff
@@ -70,7 +80,12 @@ export const events = {
   openNudgeStudyTools: mapEventType(openNudgeStudyTools),
   openStudyGuides: mapEventType(openStudyGuides),
   openUTG: mapEventType(openUTG),
-  pageFocus: mapEventType(pageFocus),
+  // debounce prevents common case of focusout immediately followed by move to background
+  // note that `unload` also triggers the visibility state change but it must be executed
+  // immediately without a debounce for the beacon to work. i don't think it will, but if
+  // this results in events representing unload followed by focusout we may have to
+  // re-visit this.
```

Comment:
> i don't think it will, but if this results in events representing unload followed by focusout we may have to re-visit this.

this is annoying to test in dev because the console closes before unload is fired, i'd like to look at the reported data from the heroku environment and make sure everything looks ok